nums/core/systems/systems.py

```diff
@@ -184,12 +184,17 @@ def warmup_func(n):
                 for _ in range(n):
                     _a = random.randint(0, 1000)
                     _b = random.randint(0, 1000)
-                    _v = self.get(r(self.put(_a), self.put(_b)))
+                    _v = self.get(
```

Comment:
Can we have this be round-robin over the devices? Including placing `_a`, `_b` on different nodes?